PROSPECTUS.md

```diff
@@ -0,0 +1,25 @@
+# Kubernetes Community Days Portland Sponsorship Opportunies
+
+_Adapted from [the Bouvet Island prospectus](https://kubernetescommunitydays.org/events/2020-bouvet-island/sponsor/)_
+
+## Post-Event Social: Contact us (Limit 1)
```

Comment:
We know the amount, it's $4000. 

I think we should also allow for 2 sponsors to split this sponsorship.

Reply:
Per Eric, increase to $5000